src/test/java/uk/gov/companieshouse/pscdataapi/api/ResourceChangedApiServiceAspectFeatureFlagEnabledITest.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.companieshouse.pscdataapi.api;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.chskafka.ChangedResource;
+import uk.gov.companieshouse.api.handler.chskafka.PrivateChangedResourceHandler;
+import uk.gov.companieshouse.api.handler.chskafka.request.PrivateChangedResourcePost;
+import uk.gov.companieshouse.api.sdk.ApiClientService;
+import uk.gov.companieshouse.pscdataapi.exceptions.ServiceUnavailableException;
+import uk.gov.companieshouse.pscdataapi.util.TestHelper;
+
+import static org.mockito.Mockito.verifyNoInteractions;
+
+@SpringBootTest
+@ActiveProfiles("feature_flag_enabled")
+class ResourceChangedApiServiceAspectFeatureFlagEnabledITest {
+    @InjectMocks
+    private ChsKafkaApiService chsKafkaApiService;
+
+    @MockBean
+    private ApiClientService apiClientService;
+
+    @Mock
+    private InternalApiClient internalApiClient;
+    @Mock
+    private PrivateChangedResourceHandler privateChangedResourceHandler;
+    @Mock
+    private PrivateChangedResourcePost changedResourcePost;
+
+    @Mock
+    private ChangedResource changedResource;
+
+    private TestHelper testHelper;
+
+    @BeforeEach
+    void setup() {
+        testHelper = new TestHelper();
+    }
+    @Test
+    void testThatAspectShouldNotProceedWhenFeatureFlagEnabled() throws ServiceUnavailableException {
```

Comment:
This test passes when ran using "mvn test", but it fails when ran on its own. Ideally we should set it up to pick up the correct environment variables even when ran on its own